testapi.pm

```diff
@@ -1144,22 +1144,30 @@ sub release_key {
 
 =head2 send_key_until_needlematch
 
-  send_key_until_needlematch($tag, $key [, $counter, $timeout]);
+  send_key_until_needlematch($tag, $key [, [$counter] | [counter => $counter], [$timeout] | [timeout => $timeout], [no_wait => $no_wait]]);
+
 
 Send specific key until needle with C<$tag> is not matched or C<$counter> is 0.
-C<$tag> can be string or C<ARRAYREF> (C<['tag1', 'tag2']>)
-Default counter is 20 steps, default timeout is 1s
+C<$tag> can be string or C<ARRAYREF> (C<['tag1', 'tag2']>).
+Default counter is 20 steps, default timeout is 1s.
+
+Specify C<$no_wait> to run the screen check as fast as possible that is
```

Comment:
I dislike copy-paste duplication. I would not even mention the "no_wait" parameter in the synopsis in L1147 but rather describe that all further arguments are forwarded to the internal check_screen call so you can give a hint that check_screen options can be used here as well :)